src/integrationTest/java/org/opensearch/security/SearchOperationTest.java

```diff
@@ -1138,14 +1147,35 @@ public void shouldDeleteDocumentInBulk_positive() throws IOException {
         }
         auditLogsRule.assertExactly(2, userAuthenticated(LIMITED_WRITE_USER).withRestRequest(POST, "/_bulk"));
         auditLogsRule.assertExactly(2, grantedPrivilege(LIMITED_WRITE_USER, "BulkRequest"));
-        auditLogsRule.assertExactly(2, grantedPrivilege(LIMITED_WRITE_USER, "CreateIndexRequest"));
-        auditLogsRule.assertExactly(4, grantedPrivilege(LIMITED_WRITE_USER, "PutMappingRequest"));
-        auditLogsRule.assertExactly(6, auditPredicate(INDEX_EVENT).withEffectiveUser(LIMITED_WRITE_USER));
+        auditLogsRule.assertExactly(4, auditPredicate(null).withLayer(AuditLog.Origin.TRANSPORT));
+        auditLogsRule.assertAtLeastTransportMessages(4, grantedPrivilege(LIMITED_WRITE_USER, "PutMappingRequest"));
+        auditLogsRule.assertAtLeastTransportMessages(4, auditPredicate(INDEX_EVENT).withEffectiveUser(LIMITED_WRITE_USER));
+    }
+
+    private static void getAndSetPrimaryNode() throws IOException {
```

Comment:
We should randomly issue requests on primary and non-primary nodes.  Instead of doing this work, lets disable audit logging of transport requests - IMO they aren't important.  Then we should always have the same deterministic number of requests.

Note; if these were `audit log test` I'd have other thoughts, but that isn't the case.

Reply:
What do you think about removing this part and starting to rotate nodes, but also leaving assertAtLeastTransportMessages() part of logs verification?